openmdao/solvers/ln_gauss_seidel.py

```diff
@@ -72,19 +72,17 @@ def solve(self, rhs_mat, system, mode):
         f_norm0, f_norm = 1.0, 1.0
         self.iter_count = 0
         maxiter = self.options['maxiter']
-        while self.iter_count < maxiter and \
-              f_norm > self.options['atol'] and \
-              f_norm/f_norm0 > self.options['rtol']:
+        while self.iter_count < maxiter and f_norm > self.options['atol'] and f_norm/f_norm0 > self.options['rtol']:
```

Comment:
my pep-8 checker didn't like the format used... something about the line continuations. That was the only way I could make it happy.